server/tournaments/index.ts

```diff
@@ -2128,16 +2128,18 @@ const commands: Chat.ChatCommands = {
 	tournamenthelp() {
 		if (!this.runBroadcast()) return;
 		this.sendReplyBox(
+			`Tournament Commands<br/>` +
 			`- create/new &lt;format>, &lt;type>, [ &lt;comma-separated arguments>]: Creates a new tournament in the current room.<br />` +
+			`- rules/banlist &lt;comma-separated arguments>: Sets the custom rules for the tournament before it has started.<br />` +
+			`- end/stop/delete: Forcibly ends the tournament in the current room.<br />` +
+			`- begin/start: Starts the tournament in the current room.<br /><br />` +
+			`<details class="readmore"><summary>More Commands</summary>` +
```

Comment:
I would make this more specific, with something like 'Tournament Configuration Commands' or smth

Reply:
Oh I didn't get a notif for this, sorry